app/scripts/templates/marketing_snippet.mustache

```diff
@@ -1,7 +1,7 @@
   {{#showSignUpMarketing}}
     <aside class="marketing desktop image-snippet default">
       <div class="marketing-spacer">
-        <span class="android-logo" />
+        <span class="android-logo"></span>
```

Comment:
Yeah, we are using html as the doctype aren't we? self-closing spans are valid only with xhtml iirc.